src/main/scala/com/fulcrumgenomics/umi/VanillaUmiConsensusCaller.scala

```diff
@@ -242,7 +252,7 @@ class VanillaUmiConsensusCaller(override val readNamePrefix: String,
   /** If a reject writer was provided, emit the reads to that writer. */
   override protected def rejectRecords(recs: Traversable[SamRecord], reason: String): Unit = {
     super.rejectRecords(recs, reason)
-    this.rejects.foreach(rej => rej ++= recs)
+    this.rejects.foreach(rej => rej ++= recs) // NB: this may not be thread-safe
```

Comment:
Nowhere does this class say that it's threadsafe, so this comment is a little odd.

Reply:
This comment was a note to our future selves that if/when we want to parallelize `CallMolecularConsensusReads`, this will need to be thread-safe.  I'll remove it.